sub.rb

```diff
@@ -0,0 +1,5 @@
+puts "subtracion method"
+def sub(a,b)
+   a+b
```

Comment:
please add the logic of handling the subtraction and include the condition for handling greater or smaller subtraction  -ve range